backend/uclapi/dashboard/views.py

```diff
@@ -147,19 +147,43 @@ def dashboard(request):
 
 @ensure_csrf_cookie
 def get_started(request):
-    logged_in = True
+    # logged_in = True
+    #
+    # try:
+    #     request.session["user_id"]
+    # except KeyError:
+    #     logged_in = False
+    #
+    # temp_token = TemporaryToken.objects.create()
+    # return render(request, 'getStarted.html', {
+    #     'initial_data': {
+    #         'temp_token': temp_token.api_token,
+    #         'logged_in': str(logged_in)
+    #     }
+    # })
+
+    from oauth.scoping import Scopes
+    s = Scopes()
 
-    try:
-        request.session["user_id"]
-    except KeyError:
-        logged_in = False
-
-    temp_token = TemporaryToken.objects.create()
-    return render(request, 'getStarted.html', {
-        'initial_data': {
-            'temp_token': temp_token.api_token,
-            'logged_in': str(logged_in)
-        }
+    page_data = {
+        "app_name": "Test app",
+        "creator": "Jane Doe",
+        "client_id": "12345.12345",
+        "state": "best-state",
+        "scopes": s.scope_dict(2),
+        "user": {
+            "full_name": "Wilhelm Klopp",
+            "cn": "zceiwhs",
+            "email": "will@ucl.ac.uk",
+            "department": "School of Management",
+            "upi": "WHSKL66"
+        },
+        "signed_data": "signed803458903459803045"
```

Comment:
Not sure how the data signing works. Is this value static for that page data or will it need to be generated at run time?

Reply:
page_data doesn't have logged_in as a key, value, which is used by the nav bar on the get started page to check whether to show a 'log in' button or not.